Src/FluentAssertions/Equivalency/CollectionMemberAssertionOptionsDecorator.cs

```diff
@@ -39,9 +39,9 @@ public OrderingRuleCollection OrderingRules
 
         public ConversionSelector ConversionSelector => inner.ConversionSelector;
 
-        public IEnumerable<IEquivalencyStep> GetUserEquivalencySteps(ConversionSelector conversionSelector)
+        public IEnumerable<IEquivalencyStep> UserEquivalencySteps
```

Comment:
Why converting this into a property?

Reply:
It used to be a property, but then became a method because of the introduction of the `ConversionSelector`. This is no longer needed, so I switched it back to be a property.